dev/langtool/meta/ckeditor.plugin-clipboard/meta.txt

```diff
@@ -6,4 +6,4 @@ copyError = A notification message shown when browser is blocking copying, and u
 cut = Toolbar button tooltip for the Cut feature.
 cutError = A notification message shown when browser is blocking cutting, and user should use the keyboard instead. %1 sequence is replaced with a keystroke that, when pressed, will perform the action.
 paste = Toolbar button tooltip for the Paste feature.
-pasteMsg = A notification message shown when browser is blocking pasting, and user should use the keyboard instead. %1 sequence is replaced with a keystroke that, when pressed, will perform the action.
+pasteDefaultMsg = A notification message shown when browser is blocking pasting, and user should use the keyboard instead. %1 sequence is replaced with a keystroke that, when pressed, will perform the action.
```

Comment:
I'd rename it simply to pasteNotification, though longer by 2 chars, it has just 2 words and reflects better what it refers to.